jenkins/node-image/node-runner.sh

```diff
@@ -27,5 +27,8 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 fi
 
+# Go to kubernetes repo
+cd kubernetes
```

Comment:
even better might be `cd "${GOPATH}/src/k8s.io/kubernetes"`, since at some point we might be enforcing running tools from a valid GOPATH.